src/controllers/organization.ts

```diff
@@ -340,18 +346,31 @@ export const getModerationRejectedController = async (
   }
 };
 
-export function getUnableToCertifyUserAsExecutiveController(
+export async function getUnableToCertifyUserAsExecutiveController(
   req: Request,
   res: Response,
   next: NextFunction,
 ) {
+  const { matches } = z
+    .object({
+      matches: z
```

Comment:
todo: matches values should be whitelisted.

Reply:
you prefer an enum there ?

Reply:
an enum should be good!